ast/ddl_test.go

```diff
@@ -62,3 +62,18 @@ func (ts *testDDLSuite) TestDDLVisitorCover(c *C) {
 		v.node.Accept(visitor1{})
 	}
 }
+
+func (ts *testDDLSuite) TestDropIndex(c *C) {
+	testCases := []NodeRestoreTestCase{
```

Comment:
please see:parser_test.go#RunTest and parser_test.go#RunRestoreTest
and see parser_test.go#L1406-L1408
```
		{"use `select`", true, "USE `select`"},
		{"use `sel``ect`", true, "USE `sel``ect`"},
		{"use select", false, "USE `select`"},
```

we hope you can test `DropIndexStmt` kile `UseStmt` in `parser_test.go`